docs/docsgen/source/technical/float8.rst

```diff
@@ -0,0 +1,105 @@
+
+.. _onnx-detail-float8:
+
+======================
+Float stored on 8 bits
+======================
+
+.. contents::
+    :local:
+
+Papers
+======
+
+Two papers have been published in 2022 to introduce floats
+stored on a byte as opposed to float 32 stored on 4 bytes.
+The float precision is much lower but the training precision
+does not suffer too much.
+
+`FP8 Formats for Deep Learning <https://arxiv.org/abs/2209.05433>`_
+from NVIDIA introduces two types following
```

Comment:
It looks like the paper was jointly authored by NVIDIA, Intel, and ARM. 
see: https://developer.nvidia.com/blog/nvidia-arm-and-intel-publish-fp8-specification-for-standardization-as-an-interchange-format-for-ai/
and: https://www.intel.com/content/www/us/en/newsroom/news/new-hardware-specs-speed-ai-software-development.html

Reply:
updated